modules/apps/info/info-api/src/main/java/com/liferay/info/fields/reader/InfoItemFieldReader.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.info.fields.reader;
+
+import com.liferay.info.fields.InfoItemField;
+import org.osgi.annotation.versioning.ProviderType;
+
+import java.util.Locale;
+
+/**
+ * @author Jürgen Kappler
+ */
+@ProviderType
+public interface InfoItemFieldReader<T> {
```

Comment:
I'm not fully convinces of calling this class FieldReader. I know I don't want it to end with "Field" because it's not really a field like InfoItemField is. 
Reader makes sense since it has a getValue() method, but maybe there are some other better options.
BTW, one alternative I considered is "FieldContributor" since we are already using the term contributor, but it's we are using it backwards "ContributorField" which I thought could cause confusion.
Thoughts?